applications/tari_base_node/src/parser.rs

```diff
@@ -433,6 +443,21 @@ impl Parser {
         });
     }
 
+    /// Function to process the ge-state-info command
+    fn process_state_info(&mut self) {
+        let mut channel = self.state_machine_info.clone();
+        self.executor.spawn(async move {
+            match channel.next().await {
+                None => {
+                    println!("Something went wrong");
```

Comment:
`None` happens when the channel closes (e.g. if the node shuts down). I wouldn't print any output in this case as it might give the user an impression of instability. The log below could be an `info!` saying that the state machine state channel closed.